### PR TITLE
Descriptors wrapper

### PR DESCRIPTION
**Pull request by bovlb**
Simple descriptors wrapper that just provides `find_similar`and `find_similar_mmr` to use in langchain.

### CODE REVIEW

File: aperturedb/Descriptors.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+
+class Descriptors(Entities):
+    """
+    Python wrapper for ApertureDB Descriptors API.
+    """
+
+    db_object = "_Descriptor"
+
+    def __init__(self, db, response=None):
```
Comment (gsaluja9) [inline aperturedb/Descriptors.py:17]: The response, in Images is to facilitate a easy wrapper creation when handling a response from a JSON query. The helper methods become easy to use then. This is obviously not interface specific, but will be a good to consider.

Comment (bovlb) [reply aperturedb/Descriptors.py]: OK.  I think that would have worked, but I don't think it's as useful for descriptors, so I took it out.

File: aperturedb/Descriptors.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        }}
+
+        if constraints is not None:
+            command["FindDescriptor"]["constraints"] = constraints.constraints
+
+        query = [command]
+        logger.debug(query)
+        blobs_in = [np.array(vector, dtype=np.float32).tobytes()]
+
+        response, blobs_out = self.db_connector.query(query, blobs_in)
```
[the remark this answers is not in the document]
Comment (bovlb) [reply aperturedb/Descriptors.py]: Done

**Review by gsaluja9 — APPROVED** [review]
Thanks. Let's merge it.

**Pull request merged**